cbor-json/shared/src/test/scala/fs2/data/cbor/CborToJsonSpec.scala

```diff
@@ -0,0 +1,161 @@
+package fs2
+package data
+package cbor
+
+import low.{items, CborItem}
+import fs2.data.json.circe._
+import fs2.data.json.Token
+import fs2.io.file.{Files, Path}
+
+import weaver._
+
+import cats.syntax.all._
+import cats.effect._
+
+import _root_.io.circe._
+import scodec.bits._
+
+object CborToJsonSpec extends SimpleIOSuite {
+
+  test("Specification tests should pass") {
+    Files[IO]
+      .readAll(Path("cbor-json/shared/src/test/resources/appendix_a.json"))
+      .through(fs2.text.utf8.decode)
+      .compile
+      .foldMonoid
+      .map(jawn.parse(_).flatMap(_.as[List[AppendixTestCase]]))
+      .rethrow
+      .map(_.zipWithIndex.collect { case (AppendixTestCase(cbor, _, _, Some(json), jsonAlt, _), idx) =>
+        (idx, cbor, json, jsonAlt)
+      })
+      .flatMap(
+        Stream
+          .emits(_)
+          .evalMap { case (idx, cbor, expected, expectedAlt) =>
+            Stream
+              .chunk(Chunk.byteVector(cbor))
+              .through(items[IO])
+              .through(fs2.data.cbor.json.decodeItems)
```

Comment:
Given the split package definition at the start of this file, this could be simplified to `json.decodeItems`. Or is the FQN intentional?